release_creation/github_client/releases.py

```diff
@@ -41,24 +42,25 @@ def _normalize_input_version(version: Version) -> Version:
     return version
 
 
-def get_latest_bundle_release(input_version: str) -> Tuple[ Version, Optional[GitRelease]]:
+def get_latest_bundle_release(input_version: str) -> Tuple[Version, bool, Optional[GitRelease]]:
     """Retrieve the latest release matching the major.minor and release stage
        semantic version if it exists. Ignores the patch version. 
 
     Args:
         input_version (str): semantic version (1.0.0.0rc, 2.3.5) to match against 
 
     Returns:
-        Tuple[ Version, Optional[GitRelease]]: A tuple of the latest release tag 
+        Tuple[ Version, is_draft, Optional[GitRelease]]: A tuple of the latest release tag, if it's in a draft state 
         and the latest release itself.
     """
     gh = get_github_client()
     target_version = Version.coerce(input_version)
     latest_version = copy.copy(target_version)
     latest_version = _normalize_input_version(latest_version)
     repo = gh.get_repo(_GH_BUNDLE_REPO)
-    releases = repo.get_releases()
+    releases = repo.get_releases()  # must have push access to get draft releases
```

Comment:
I think we can assert that we have push access simply by running:
`repo.get_collaborator_permission(gh.get_user().login)`

Reply:
After looking more into what push access means, everyone at dbt Labs should have that access.  Since we are the only ones who should be (or even can) submit this, this check is not necesary.